<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-02-06T09:58:41 IST" name="Default suite" finished-at="2024-02-06T09:59:03 IST" duration-ms="21707">
    <groups>
    </groups>
    <test started-at="2024-02-06T09:58:41 IST" name="Default test" finished-at="2024-02-06T09:59:03 IST" duration-ms="21707">
      <class name="Testcase.MakeMyTripSearch">
        <test-method is-config="true" signature="lauchBrowser()[pri:0, instance:Testcase.MakeMyTripSearch@5fe94a96]" started-at="2024-02-06T09:58:38 IST" name="lauchBrowser" finished-at="2024-02-06T09:58:41 IST" duration-ms="2761" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- lauchBrowser -->
        <test-method is-config="true" signature="EnterUrlAndMaximize()[pri:0, instance:Testcase.MakeMyTripSearch@5fe94a96]" started-at="2024-02-06T09:58:41 IST" name="EnterUrlAndMaximize" finished-at="2024-02-06T09:58:44 IST" duration-ms="2762" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- EnterUrlAndMaximize -->
        <test-method signature="ValidSearch()[pri:0, instance:Testcase.MakeMyTripSearch@5fe94a96]" started-at="2024-02-06T09:58:44 IST" name="ValidSearch" finished-at="2024-02-06T09:59:00 IST" duration-ms="16397" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Flights from Chennai to  Pune] but found [Flights from Chennai to Pune]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Flights from Chennai to  Pune] but found [Flights from Chennai to Pune]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at Testcase.MakeMyTripSearch.ValidSearch(MakeMyTripSearch.java:75)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ValidSearch -->
        <test-method signature="ValidatesameCityError()[pri:1, instance:Testcase.MakeMyTripSearch@5fe94a96]" started-at="2024-02-06T09:59:00 IST" name="ValidatesameCityError" finished-at="2024-02-06T09:59:03 IST" duration-ms="2500" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ValidatesameCityError -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:Testcase.MakeMyTripSearch@5fe94a96]" started-at="2024-02-06T09:59:03 IST" name="teardown" finished-at="2024-02-06T09:59:03 IST" duration-ms="455" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- Testcase.MakeMyTripSearch -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
